lib/babe/mocks/mock_state.go

```diff
@@ -0,0 +1,839 @@
+// Code generated by MockGen. DO NOT EDIT.
+// Source: ./lib/babe/state.go
+
+// Package mock_babe is a generated GoMock package.
+package mocks
+
+import (
+	big "math/big"
+	reflect "reflect"
+	time "time"
+
+	types "github.com/ChainSafe/gossamer/dot/types"
+	common "github.com/ChainSafe/gossamer/lib/common"
+	runtime "github.com/ChainSafe/gossamer/lib/runtime"
+	storage "github.com/ChainSafe/gossamer/lib/runtime/storage"
+	transaction "github.com/ChainSafe/gossamer/lib/transaction"
+	gomock "github.com/golang/mock/gomock"
```

Comment:
All these don't need to be aliased. Girlfriend is coming to get me, my review will end here for now :laughing: